neuralmonkey/learning_utils.py

```diff
@@ -209,6 +209,13 @@ def training_loop(tf_manager: TensorFlowManager,
                                         tf_manager.best_score_batch),
                                 color='blue')
 
+                            # store also graph parts
+                            all_coders = set.union(
+                                *[rnr.all_coders for rnr in runners])
```

Comment:
Runnery a trainery si sbírají všechny model parts rekurzivně. Potřebují to k tomu, aby mohli TF manageru říct, kam chtějí feedovat.

Sbíral bych spíš jenom z traineru. Smysl ukládat má jenom to, co se trénuje. Jinak, pokud tam mají být i ty části z runnerů, tak bych se s tím nepáral a do té list comprehension bych napsal `runners + [trainer]`.